powerVCF.psm1

```diff
@@ -2580,6 +2580,123 @@ Export-ModuleMember -Function Get-VCFvROPs
 
 ######### End vRealize Suite Operations ##########
 
+######### Start Federation Management ##########
+
+Function Get-SDDCFederation {
```

Comment:
All other cmdlets have VCF in the name to distinguish them from non VCF cmdlets.

Reply:
Same for the other functions below